.travis.yml

```diff
@@ -1,21 +1,16 @@
-addons:
-    postgresql: "9.3"
-
+sudo: false
 language: python
+cache: pip
 
-python:
-  - "2.7"
-
-# todo: find a way to run 1.4 tests as well. That requires jmbo>2.0.0. I don't
-# know how to do that yet.
 env:
-  - DJANGO=Django==1.6.8
+  - TOXENV=django111-py27
```

Comment:
Think we need a 35 env too.

Reply:
Sure. I'll add. Just an FYI that those tests will fail at present though